en/docs/get-started/quick-start-guide/integration-qsg.md

```diff
@@ -1,17 +1,17 @@
 # Integration Quick Start Guide
 
-Let's get started with WSO2 API Manager by running a simple integration use case in your local environment. 
+Let's get started with WSO2 Micro Integrator by running a simple integration use case in your local environment. 
 
 ## Before you begin
 
-1. Go to the [website](https://wso2.com/api-management/) to download WSO2 API Manager and WSO2 Integration Studio. 
+1. Go to the [website](https://wso2.com/api-management/) to download WSO2 Micro Integrator and WSO2 Integration Studio. 
 
     When you click **Download**, the installation options will be listed. For this quick start, you can either download and run the **installer**, or use the **binary** file.
 
     !!! Info
         For more information, see the [installation instructions]({{base_path}}/install-and-setup/install-and-setup-overview/#installing_1).
 
-2. Download the [sample files](https://github.com/wso2/docs-ei/blob/master/en/micro-integrator/docs/assets/attach/quick-start-guide/MI_QSG_HOME-JDK11.zip). From this point onwards, let's refer to this folder as `<MI_QSG_HOME>`.
+2. Download the [sample files]({{base_path}}/assets/attachments/tutorial/MI_QSG_HOME.zip). From this point onwards, let's refer to this folder as `<MI_QSG_HOME>`.
```

Comment:
File names should only have lowercase letters and dashes. If there are uppercase, the link will break.

Reply:
Fixed